src/components/FullscreenGrid/ContentShell.spec.tsx

```diff
@@ -182,4 +196,11 @@ describe(__filename, () => {
       fullscreenGridActions.toggleAltSidePanel(),
     );
   });
+
+  it('renders the main content panel with a location key', () => {
+    const key = 'locationKey';
+    const root = render({ location: createFakeLocation({ key }) });
+
+    expect(root.find(`.${styles.content}`).key()).toEqual(key);
```

Comment:
I didn't even know `key()` was a thing! A guess it's rare to need to assert on a `key` in a test.

Reply:
Heh, yeah, I learned about it recently too. This kind of test relies on trusting the React API but key behavior seems stable enough. It's features like that where having some functional test coverage would be nice.